src/richie/apps/core/tests/utils.py

```diff
@@ -5,6 +5,58 @@
 import os
 import random
 
+from cms.test_utils.testcases import CMSTestCase
+
+
+class CMSPluginTestCase(CMSTestCase):
+    """
+    Enriched CMS test case object to include useful stuff about plugin
+    rendering.
+
+    Note:
+        CMSTestCase mixin from DjangoCMS cause unknow issue with transactions
+        on some tests and lead to necessity to decorate involved tests with
+        ``django.db.transaction`` (or to use it as a context manager) which is
+        undesired.
+
+        We should change this mixin to not inherit anymore from CMSTestCase
+        and try to build ourself a correct context.
```

Comment:
Maybe change the whole note to:
```
Note:
    The CMSTestCase mixin from DjangoCMS is making queries in its "tearDown" method which
    causes database transaction errors with tests that have already generated a database error.

    A temporary workaround is to decorate such tests with ``django.db.transaction`` (or to use it as a context manager) so that the test has its own transaction that is rolled back before the TearDown method sends its queries to the database.
```